tests/storage/storage_migration/utils.py

```
@@ -34,9 +39,55 @@ def verify_vms_boot_time_after_storage_migration(
+            '| grep -iE "shutdown|reboot|restart|power|kill" | tail -n 100'
+            "'"
+        )
+
+    try:
+        output = run_ssh_commands(host=vm.ssh_exec, commands=cmd, wait_timeout=TIMEOUT_2MIN)[0]
+        write_to_file(
+            base_directory=base_dir,
+            file_name=f"{vm.name}_reboot_events.txt",
+            content=output,
+        )
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)